src/aiperf/credit/sticky_router.py

```
@@ -347,6 +347,24 @@ async def wait_for_workers(self, timeout: float) -> None:
+    async def _fire_virtual_return(self, credit_return: CreditReturn) -> None:
+        """Deliver a synthesized no_request CreditReturn to the return consumer.
+
+        Runs as a detached task (scheduled via ``execute_async``). Exceptions are
+        logged here rather than surfacing only through asyncio's default
+        "Task exception was never retrieved" handler -- otherwise a failure on the
+        spawn-dispatch path (``intercept``) would become an opaque phase-timeout
+        hang with no structured log.
+        """
+        try:
+            await self._on_return_callback("", credit_return)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)